templates/email/default/_email_comment_list.txt

```diff
@@ -1,7 +1,7 @@
 [% FOR row IN data -%]
 [% row.item_name _ ' : ' IF row.item_name AND NOT row.item_anonymous -%]
 [% '(' _ cobrand.prettify_dt(row.confirmed) _ ') ' IF cobrand.include_time_in_update_alerts -%]
-[% row.item_text %]
+[% row.item_text | strip_html %]
```

Comment:
Hmm, strip_html won't make much sense if e.g. they've used `a` or `img`. Guess it needs some HTML to text parser. Googling gives HTML::FormatText::WithLinks which looking at the author, umm, we can probably use that for starters ;) Doesn't look like it handles lists though. And images presumably need to become alt text...
Alternatively, perhaps it could say this update had HTML, please view on the site.

Reply:
Oh, this comment still applies - if we want to get it live, perhaps the message I suggest? Or that module?

Reply:
Sigh, overlooked this! Will take a look at that module next week.